### PR TITLE
[docs] document 'version' command 📃⚡🚀

### PR DESCRIPTION
**Pull request by valeriaconde**
**Describe the pull request**

- #### What does your PR fix? 
  Documentation 😊❤😍😜

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
✌ n/a

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
obvs ✔

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
not adding a port 😢😢😢

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: docs/commands/version.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Synopsis
+
+```no-highlight
+vcpkg version
+```
+
+## Description
+
+Displays the version of vcpkg installed.
```
Comment (vicroms) [inline docs/commands/version.md:13]:
```suggestion
Displays the version of the vcpkg executable.

```

We are not actually "installed" so I would avoid that word.

**Review by vicroms — CHANGES_REQUESTED** [review]

File: docs/commands/version.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# vcpkg version
+
+**The latest version of this documentation is available on [GitHub](https://github.com/Microsoft/vcpkg/tree/master/docs/commands/integrate.md).**
```
Comment (ras0219-msft) [inline docs/commands/version.md:3]:
```suggestion
**The latest version of this documentation is available on [GitHub](https://github.com/Microsoft/vcpkg/tree/master/docs/commands/version.md).**
```

Comment (valeriaconde) [reply docs/commands/version.md]: Thank you 😦

File: docs/commands/version.md (hunk 1 above)
Comment (valeriaconde) [reply docs/commands/version.md]: Thanks! 😄😄

**Review by vicroms — APPROVED** [review]

**Pull request merged**